enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/load_balancing/GetServersProcedureV1.java

```diff
@@ -50,47 +50,56 @@
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Stream.concat;
 import static org.neo4j.causalclustering.core.CausalClusteringSettings.cluster_allow_reads_on_followers;
+import static org.neo4j.causalclustering.load_balancing.Role.READ;
+import static org.neo4j.causalclustering.load_balancing.Role.ROUTE;
+import static org.neo4j.causalclustering.load_balancing.Role.WRITE;
+import static org.neo4j.causalclustering.load_balancing.GetServersParameters.SERVERS;
+import static org.neo4j.causalclustering.load_balancing.GetServersParameters.TTL;
+import static org.neo4j.causalclustering.load_balancing.ProcedureNames.GET_SERVERS_V1;
 import static org.neo4j.kernel.api.proc.ProcedureSignature.procedureSignature;
 
-/*
-C: RUN "CALL dbms.cluster.routing.getServers" {}
-   PULL_ALL
-S: SUCCESS {"fields": ["ttl", "servers"]}
-   RECORD [9223372036854775807, [
-{"role": "WRITE", "addresses": ["127.0.0.1:9001"]},
-{"role": "READ", "addresses": ["127.0.0.1:9002", "127.0.0.1:9003"]},
-{"role": "ROUTE", "addresses": ["127.0.0.1:9001", "127.0.0.1:9002", "127.0.0.1:9003"]}
-]]
+/**
+ * Returns endpoints and their capabilities.
+ *
+ * TODO: Detail signature (input and output).
```

Comment:
I removed it on purpose because it was very messy and didn't seem to fit as a javadoc for the class. I will introduce a better description when I get around to it for the development of the V2. This does not imply a change of the results format itself.